packages/unmock-core/src/service/interfaces.ts

```diff
@@ -32,6 +42,11 @@ export interface IServiceInput {
   name: string;
 }
 
+export interface IOASMatcher {
```

Comment:
Does this need an interface if it's only used internally by the `Service`?

Reply:
Not really - was mostly thinking it could be useful if we'd like to extend it for other formats such as `RAML`, but it's definitely not needed now. I'll remove it then.